### DISCUSSION TITLE
document cover the completion information

### DISCUSSION THREAD

**Discussion by OnlyWick** [mrcjkb/rustaceanvim#517] [Q&A]
How can it be avoided?
<img width="1301" alt="image" src="https://github.com/user-attachments/assets/a28c2279-d280-483e-9c95-bf58cb3e43a0">

**Comment by mrcjkb** [comment]
hey 👋 

rustaceanvim doesn't implement completion. You're better off searching or asking in the discussions/issues of the completion plugin you're using.

**Reply by OnlyWick** [reply]
That's very weird, the other completions don't have this issue. Okay, Thank you for your reply.
